agentendpoint/patch_task.go

```diff
@@ -45,11 +45,11 @@ type patchTask struct {
 	lastProgressState map[agentendpointpb.ApplyPatchesTaskProgress_State]time.Time
 	state             *taskState
 
-	TaskID      string
-	Task        *applyPatchesTask
-	StartedAt   time.Time `json:",omitempty"`
-	PatchStep   patchStep `json:",omitempty"`
-	RebootCount int
+	TaskID               string
+	Task                 *applyPatchesTask
+	StartedAt            time.Time `json:",omitempty"`
+	PatchStep            patchStep `json:",omitempty"`
+	PostPatchRebootCount int
```

Comment:
I'm a bit scared that we might get prePatch reboots loop, do you think it might be a good idea to leave RebootCount counter and break the loop of prePartch reboots if the RebootCount counter is bigger than 3 or 5 smth like that?